torch/ao/quantization/quantize_fx.py

```diff
@@ -544,7 +696,16 @@ def convert_to_reference_fx(
     Example::
 
         # prepared_model: the model after prepare_fx/prepare_qat_fx and calibration/training
-        reference_model = convert_to_reference_fx(prepared_model)
+        # convert_to_reference_fx converts observer/fake_quant modules in the model to
+        # quantize and dequantize operators, and produces a reference quantized model
+        # see https://github.com/pytorch/rfcs/blob/master/RFC-0019-Extending-PyTorch-Quantization-to-Custom-Backends.md
+        # for more information about reference quantized model
+        # The reference quantized model is a standard format for a quantized model provided
+        # by fx graph mode quantization, it can be further lowered to run on the target
+        # hardware
```

Comment:
Should we move this paragraph to the main docstring instead of under `Example::`?

Reply:
makes sense, will merge with the main docstring